pkg/state/state.go

```diff
@@ -434,15 +435,12 @@ var DefaultFetchOutputDirTemplate = filepath.Join(
 	"{{ or .Release.Version \"latest\" }}",
 )
 
-func (st *HelmState) ApplyOverrides(spec *ReleaseSpec) {
-	if st.OverrideKubeContext != "" {
-		spec.KubeContext = st.OverrideKubeContext
-	}
-	if st.OverrideNamespace != "" {
-		spec.Namespace = st.OverrideNamespace
-	}
-
+func (st *HelmState) ReFormatNeeds(spec *ReleaseSpec) []string {
```

Comment:
```suggestion
func (st *HelmState) reformatNeeds(spec *ReleaseSpec) []string {
```

nit but we should make this a private function as it's not used elsewhere! I also think we can just treat "reformat" an independent word rather than a compound word of "re-format" so let's use "Reformat" instead of "ReFormat"!